packages/vitest-environment-nuxt/src/env/jsdom.ts

```diff
@@ -0,0 +1,44 @@
+import { importModule } from 'local-pkg'
+import { EnvironmentNuxt } from '../types'
+
+export default <EnvironmentNuxt> async function (global, { jsdom = {} }) {
```

Comment:
Is this a copy from the JSDOM env from Vitest? If so, couldn't we just import and use this from Vitest?

Reply:
We can't use it directly from Vitest, because we need to return window object here.